website/docs/observability/logging/index.md

```diff
@@ -10,3 +10,13 @@ Containers are grouped as Pods within a Kubernetes cluster and are scheduled to
 In this lab, we'll see 
  - How to enable EKS Control Plane logs and verify it in the Amazon CloudWatch
  - How to setup the logging agent (Fluent Bit) to stream the Pod logs to Amazon CloudWatch
+
+
+:::tip Before you start
```

Comment:
This is not necessary since it isn't a lab

Reply:
Will remove the note